packages/dotcom-server-navigation/src/navigation.ts

```diff
@@ -56,7 +58,17 @@ export class Navigation {
   async getMenusData(): Promise<TNavMenus> {
     // initialPromise does not return data but must resolve before `getData` can be called
     await this.initialPromise
-    return this.poller.getData()
+
+    try {
+      this.menuData = this.poller.getData()
+    } catch (error) {
+      // getData throws if the most recent fetch resulted in an error.
+      // In that case, continue to use the latest navigation we have available.
```

Comment:
**request:** Please could we amend the comment to include that this is a temporary solution until there is a solution to the uncaught error issue?